packages/cw-orch-traits/Cargo.toml

```diff
@@ -13,5 +13,5 @@ readme = "README.md"
 
 [dependencies]
 cw-orch-core = { workspace = true }
-prost-types = "0.11.0"
-prost = "0.11.0"
+prost-types = { version = "0.11.9" }
```

Comment:
use workspace here as well?

Reply:
No, because it's not compatible with osmosis-test-tube. Will add a comment there